emily/handler/tests/integration/withdrawal.rs

```diff
@@ -19,6 +19,12 @@ const BLOCK_HASH: &'static str = "TEST_BLOCK_HASH";
 const BLOCK_HEIGHT: u64 = 0;
 const INITIAL_WITHDRAWAL_STATUS_MESSAGE: &'static str = "Just received withdrawal";
 
+#[derive(Debug)]
+enum WithdrawalError {
```

Comment:
Same Q as above

Reply:
[89d24c4](https://github.com/stacks-sbtc/sbtc/pull/1586/commits/89d24c4e09852299089583ff891fbd22e9e1f283)

Reply:
Hmm, don't think so, just did search for WithdrawalError and DepositError and nothing interesting. Maybe I updated it in some other commit....

Reply:
Just double-checked, I don't see any of this structs in diff between this branch and main. Do you see them?